shop/sell.php

```diff
@@ -505,6 +509,7 @@ function KeyDownCheck(KeyID)
 		$idplus++;
 	}
 }
+// What is this?

```

Comment:
What is what?

Reply:
Oh yeah, it was the `$pass_to_javascript`, it was giving me errors until i found out it's actual purpose. You can remove it.

Reply:
It's your PR, so you have to be the one to remove it haha